.github/workflows/dispatch-release-performance-check.yml

```
@@ -35,7 +34,6 @@ jobs:
           event-type: performance-test
           client-payload: '{"pr": "${{ github.event.issue.number  }}"}'
```

Maybe you'll want to remove a space here too
```suggestion
          client-payload: '{"pr": "${{ github.event.issue.number }}"}'
```